_topic_map.yml

```diff
@@ -826,18 +826,24 @@ Topics:
 - Name: What huge pages do and how they are consumed by apps
   File: what-huge-pages-do-and-how-they-are-consumed-by-apps
 ---
-Name: Disaster recovery
-Dir: disaster_recovery
+# TODO: don't forget about setting up redirects
+Name: Backup and restore
```

Comment:
If we're nesting these. I think it should be Disaster Recovery as the top level item with Backup and Restore under that. Maybe get a second opinion before making this change though.